app/assets/javascripts/modules/users/components/signup-form.js

```diff
@@ -0,0 +1,36 @@
+import BaseComponent from '~/base/component';
+
+const PASSWORD_FIELD = '#user_password';
+const CONFIRMATION_PASSWORD_FIELD = '#user_password_confirmation';
+const SUBMIT_BUTTON = '#submit_btn';
+
+// UsersPasswordForm component that handles user password form
+// interactions.
+class UsersSignUpForm extends BaseComponent {
+  elements() {
+    this.$Password = this.$el.find(PASSWORD_FIELD);
```

Comment:
Could you rename variable to lower case ? (e.g. `this.$Password` should be `this.$password`).